modules/openapi-generator/src/main/resources/kotlin-server/libraries/ktor/build.gradle.mustache

```diff
@@ -1,15 +1,15 @@
 group '{{groupId}}'
 version '{{artifactVersion}}'
 
-task wrapper(type: Wrapper) {
-    gradleVersion = '4.3'
-    distributionUrl = "https://services.gradle.org/distributions/gradle-$gradleVersion-all.zip"
+wrapper {
+gradleVersion = '4.9'
+distributionUrl = "https://services.gradle.org/distributions/gradle-$gradleVersion-all.zip"
 }
 
 buildscript {
-    ext.kotlin_version = '1.2.10'
-    ext.ktor_version = '0.9.1-alpha-9'
-    ext.shadow_version = '2.0.2'
+    ext.kotlin_version = '1.3.21'
+ext.ktor_version = '1.1.3'
+ext.shadow_version = '2.0.3'
```

Comment:
should line 11 and 12 have 4 leading spaces same as line 10?

Reply:
thanks for the review. i update the PR to fix the indentation